tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example_raw_scalars/util.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utilities for the plugin backend."""
+
+import os
+
+
+def can_serve_from_static(path):
```

Comment:
Acknowledged that this is in its own module for testing purposes, but
this has two notable drawbacks: (a) it’s harder to follow the plugin
code than if this were inlined or at least a module-private function;
and (b) the test-time imports are hard to reason about, as [discussed
earlier][rc]. It’s not clear to me what the best solution is; do you see
a way to better address these two concerns?

[rc]: https://github.com/tensorflow/tensorboard/pull/3137#discussion_r365884435